{{cookiecutter.project_name}}/docker-compose.yml

```diff
@@ -10,7 +10,7 @@
 version: "3.6"
 services:
   db:
-    image: "postgres:9.6-alpine"
+    image: "postgres:13-alpine"
```

Comment:
Great idea! 😆

Reply:
BTW, PostgreSQL v14 Beta 1 is released already :eyes: 

@sobolevn , do you know why there is a problem with DB? 

> 2021/05/27 09:14:57 Problem with dial: dial tcp: lookup db on 127.0.0.11:53: server misbehaving. Sleeping 1s

Reply:
Nope, I am still using `9.6`